scripts/legacy-db-migration/revert/revert_target_db.sql

```diff
@@ -0,0 +1,21 @@
+-- Delete imported data
+DELETE FROM error_logs;
+DELETE FROM individual_requests;
+DELETE FROM bulk_submissions;
+DELETE FROM service_requests;
+DELETE FROM bulk_customer_applications;
+DELETE FROM bulk_customers;
+
+-- Reset sequences
+ALTER SEQUENCE BULK_CUST_APP_SEQ RESTART WITH 1;
```

Comment:
restart with 1, is this correct?

Reply:
Yes.  The revert script removes all data from the tables that these sequences correspond to, so they should be reset to 1.

For reference, when the sequences are initially set up in the V0001__init flyway script they set to start at 1 as well.